docs/_config.yml

```diff
@@ -18,6 +18,8 @@
 remote_theme: pmarsceill/just-the-docs
 
 aux_links:
+  "Spark Rapids User Guide":
+    - "//docs.nvidia.com/spark-rapids/user-guide/latest/index.html"
   "Spark Rapids Plugin on Github":
```

Comment:
We should be using RAPIDS Accelerator for Apache Spark (or RAPIDS Accelerator if needs to be shortend) rather than Spark Rapids.

Reply:
OK, updated